From dea7584314f0d060e6d5d818b1c00c7053f53777 Mon Sep 17 00:00:00 2001
From: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
Date: Mon, 13 Nov 2017 11:36:10 +0900
Subject: [PATCH 03/23] Revert "arm64: dts: r8a7796: Add MFIS device node"

This reverts commit f249493ec3e2 ("arm64: dts: r8a7796: Add MFIS
device node")

Signed-off-by: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a7796.dtsi | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796.dtsi b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
index 552892b..539bd38 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
@@ -758,14 +758,6 @@
 				     "renesas,mfis-lock";
 			reg = <0 0xe6260000 0 0x1000>;
 			#hwlock-cells = <1>;
-
-			mfis_as: mfis-as {
-				compatible = "renesas,mfis-as-r8a7796",
-					     "renesas,mfis-as";
-				interrupts = <GIC_SPI 180 IRQ_TYPE_LEVEL_HIGH>;
-				interrupt-names = "eicr0";
-				renesas,mfis-ch = <0>;
-			};
 		};
 
 		pfc: pin-controller@e6060000 {
-- 
1.9.1

